ci/config_test_cases.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+TEST_CASES = {
+    "spleen_ct_segmentation": {
+        "dataset_size": 10,
+        "input_shape": (64, 64, 64),
+        "check_determinism": True,
+        "train_override": {
```

Comment:
May I know why overriding these keys?

Thanks.

Reply:
Hi @Nic-Ma , it's to reduce the resources used. Otherwise we may meet shared memory error as in: https://github.com/Project-MONAI/model-zoo/actions/runs/4832703584/jobs/8611811308